v2/internal/controllers/crd_networking_forwardingrule_test.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+
+	"github.com/Azure/azure-service-operator/v2/api/network/v1api20220701"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+	"github.com/Azure/azure-service-operator/v2/internal/util/to"
+)
+
+func Test_Networking_ForwardingRuleSet_CRUD(t *testing.T) {
+	t.Parallel()
+
+	tc := globalTestContext.ForTest(t)
+
+	rg := tc.CreateTestResourceGroupAndWait()
+
+	vnet := newVNet(tc, testcommon.AsOwner(rg), []string{"10.0.0.0/8"})
+	resolver := newDnsResolver(tc, rg, vnet)
+	subnet := newSubnet(tc, vnet, "10.225.0.0/28")
+	outboundEP := newDnsResolversOutboundEndpoint(tc, resolver, subnet)
+
+	tc.CreateResourcesAndWait(vnet, subnet, resolver, outboundEP)
+
+	ruleSet := &v1api20220701.DnsForwardingRuleset{
```

Comment:
minor: Maybe alias this to just `network`? It's easier to read than `v1api20220701`